<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>921</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #E8D5B5;</string>
   </property>
   <widget class="QWidget" name="restaurant_sidebar" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>88</y>
      <width>142</width>
      <height>440</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#343131;border-top-right-radius:20;border-bottom-right-radius:20</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_25">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_26">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_11" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>6</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="restaurant_photo_label">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius:40;border:1px solid black; color: #FABC3F;</string>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>7</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="restaurant_name_label">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:#FABC3F</string>
         </property>
         <property name="text">
          <string>Name Restaurant</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>78</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_27">
       <property name="leftMargin">
        <number>30</number>
       </property>
       <item alignment="Qt::AlignLeft">
        <widget class="QPushButton" name="restaurant_info_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:transparent; color:#FABC3F;text-align:left</string>
         </property>
         <property name="text">
          <string>Info</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../image/sidebar_info.png</normaloff>../image/sidebar_info.png</iconset>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QPushButton" name="restaurant_menu_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:transparent; color:#FABC3F;text-align:left</string>
         </property>
         <property name="text">
          <string> Menu</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../image/sidebar_menu.png</normaloff>../image/sidebar_menu.png</iconset>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QPushButton" name="restaurant_review_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background:transparent; color:#FABC3F;text-align:left</string>
         </property>
         <property name="text">
          <string>Review</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../image/sidebar_review.png</normaloff>../image/sidebar_review.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>77</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>781</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="menu_page">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>752</width>
        <height>440</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="menu_verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_22">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../image/ic_filter24.png</normaloff>../image/ic_filter24.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>20</height>
            </size>
           </property>
           <property name="shortcut">
            <string/>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>8</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #FABC3F;background-color: #343131;</string>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>8</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Edit&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #FABC3F;background-color: #343131;</string>
           </property>
           <property name="text">
            <string>Edit</string>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_9">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>8</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Edit&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #FABC3F;background-color: #343131;</string>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>750</width>
           <height>400</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>167772</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
background-color: rgb(255, 243, 228);</string>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>150</number>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>30</number>
         </attribute>
         <attribute name="verticalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>FOOD</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="background">
           <color>
            <red>52</red>
            <green>49</green>
            <blue>49</blue>
           </color>
          </property>
          <property name="foreground">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>250</red>
             <green>188</green>
             <blue>63</blue>
            </color>
           </brush>
          </property>
         </column>
         <column>
          <property name="text">
           <string>RATE</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="background">
           <color>
            <red>52</red>
            <green>49</green>
            <blue>49</blue>
           </color>
          </property>
          <property name="foreground">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>250</red>
             <green>188</green>
             <blue>63</blue>
            </color>
           </brush>
          </property>
         </column>
         <column>
          <property name="text">
           <string>PRICE</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="background">
           <color>
            <red>52</red>
            <green>49</green>
            <blue>49</blue>
           </color>
          </property>
          <property name="foreground">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>250</red>
             <green>188</green>
             <blue>63</blue>
            </color>
           </brush>
          </property>
         </column>
         <column>
          <property name="text">
           <string>DESCRIPTION</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="background">
           <color>
            <red>52</red>
            <green>49</green>
            <blue>49</blue>
           </color>
          </property>
          <property name="foreground">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>250</red>
             <green>188</green>
             <blue>63</blue>
            </color>
           </brush>
          </property>
         </column>
         <column>
          <property name="text">
           <string>REVIEW</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
          <property name="background">
           <color>
            <red>52</red>
            <green>49</green>
            <blue>49</blue>
           </color>
          </property>
          <property name="foreground">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>250</red>
             <green>188</green>
             <blue>63</blue>
            </color>
           </brush>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Qss/icons/_icons.qrc"/>
 </resources>
 <connections/>
</ui>
